runtime/compiler/net/MessageTypes.hpp

```diff
@@ -207,6 +207,7 @@ enum MessageType : uint16_t
 
    // For static TR::CompilationInfo methods
    CompInfo_isCompiled, // 171
+   CompInfo_getPCIfCompiled,
```

Comment:
Since a new message was introduced we need to increment the minor version info here:
https://github.com/eclipse/openj9/blob/master/runtime/compiler/net/CommunicationStream.hpp#L105

Reply:
Ah right, thanks